pkg/platform/abstract/platform.go

```diff
@@ -442,7 +444,19 @@ func (ap *Platform) GetProcessorLogsAndBriefError(scanner *bufio.Scanner) (strin
 
 			// when it is unstructured just add the log as a text
 			formattedProcessorLogs += rawLogLine + "\n"
-			*briefErrorsArray = append(*briefErrorsArray, rawLogLine)
+
+			// if there's a panic or call stack is printed,
+			// stop appending raw log lines to the briefErrorsMessage (unnecessary information from now on)
+			// this information can still be found in the full log
+			if strings.HasPrefix(rawLogLine, "panic: Wrapper") ||
+				strings.HasPrefix(rawLogLine, "Call stack:") {
+				stopWritingRawLinesToBriefErrorsMessage = true
+			}
+
+			if !stopWritingRawLinesToBriefErrorsMessage {
+				*briefErrorsArray = append(*briefErrorsArray, rawLogLine)
+			}
```

Comment:
Since there is no logic to turn `stopWritingRawLinesToBriefErrorsMessage` back to `false` after it's set to `true`
Why do you continue the scan loop? break

Reply:
Notice there's a distinction between _raw_ log line, and _structured_ log line.
Raw log lines are not structured because they weren't printed by a logger (e.g lines printed by `panic`..)
The scan loop continues in order to get all the _structured_ log lines that may be afterwards in the logs

Reply:
oops missed that